src/App/Service/Model/Contribution.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace Console\App\Service\Model;
+
+use Datetime;
+
+/**
+ * @see ReviewContributionsByOrganization
+ */
+class Contribution
+{
+    /**
+     * @var Datetime
```

Comment:
```suggestion
     * @var Datetime|null
```

Reply:
I will change this to set attributes in construct because these attributes are required